lib/backburner/platform.ts

```diff
@@ -9,29 +9,29 @@ export interface IPlatform {
 const SET_TIMEOUT = setTimeout;
 const NOOP = () => {};
 
-export function buildPlatform(flush: () => void): IPlatform {
-  let next;
-  let clearNext = NOOP;
+export function buildNext(flush: () => void): () => void {
+  if (typeof Promise === 'function') {
```

Comment:
Let's leave a quick inline comment, as to why this order was choosen

Reply:
Added a comment...